src/sorcha_community_utils/activity/lsst_comet/model.py

```diff
@@ -0,0 +1,129 @@
+__all__ = ["Comet"]
+
+import numpy as np
+
+try:
+    import astropy.units as u
+except ImportError:
+    u = None
+
+from .phase import phase_HalleyMarcus
+
+# TODO: Change this to use the Astropy definition
+AU_IN_CM = 14959787070000
+
+
+class Comet:
+    """LSST comet model.
+
+    The goal of this model is to make a template comet based on ``H``
+    magnitude, which is the main brightness parameter in the LSST
+    Metrics Analysis Framework.  Using this value enables easy object
+    cloning.
+
+
+    Parameters
+    ----------
+    R : float
+        Absolute magnitude or radius (km) of the nucleus.
```

Comment:
I don't actually think R is needed I think it's not used anywhere because we got rid of all the nucleus bits

Reply:
Removed